proposals/Exceptions.md

```diff
@@ -307,14 +307,30 @@ or defined:
 ### Exception section
 
 The `exception` section is the named section 'exception' with id 13. The
-exception section must occur before the code section.  The exception section
-declares exception types using exception type signatures.
+exception section may occur anywhere after the import section and before the
+export section. There may be at most one exception section.  The exception
+section declares exception types using exception type signatures.
 
 | Field | Type | Description |
 |-------|------|-------------|
 | count | `varuint32` | count of the number of exceptions to follow |
 | sig | `except_type*` | The type signature of the data fields for each exception |
 
+#### Rationale
+
+The exception is assigned the number 13 because this is the next available
+identifier.
+The [spec](https://webassembly.github.io/spec/binary/modules.html#sections)
+currently shows numbers 0 through 11 are assigned. Section 12 appears to be
+reserved for the name section.
```

Comment:
Yeah, I meant where is the name sections specified as section 12. I'd say that v8 is wrong here -- the spec says nothing about the name section being numbered.